internal/test_repl_info_replica.go

```diff
@@ -10,14 +10,16 @@ import (
 )
 
 func testReplInfoReplica(stageHarness *testerutils.StageHarness) error {
+	logger := stageHarness.Logger
+
 	listener, err := net.Listen("tcp", ":6379")
 	if err != nil {
-		fmt.Println("Error starting TCP server:", err)
+		logger.Errorf("Error starting TCP server: %v", err)
+		logFriendlyBindError(logger, err)
 		return err
```

Comment:
```suggestion
		return fmt.Errorf("Error starting TCP server: %v", err)
```

(if we do this and remove the logger.Efforf above, we won't have the error logged twice)